ISeriesProvider/DB2iSeriesTools.cs

```diff
@@ -9,200 +9,454 @@
 
 namespace LinqToDB.DataProvider.DB2iSeries

```

Comment:
Rewritten most of this, regions are self explanatory, you can start a review to proceed